javascript-forms/form-events.js

```diff
@@ -0,0 +1,18 @@
+function handleFocus(event) {
+  console.log('focus event was fired');
+  console.log(event.target.name);
+}
+function handleBlur(event) {
+  console.log('blur event was fired');
+  console.log(event.target.name);
+}
+function handleInput(event) {
+  console.log(event.target.name);
+  console.log(event.target.value);
+}
+var username = document.querySelector('#user-name');
+var useremail = document.querySelector('#user-email');
+var message = document.querySelector('#user-message');
+username.addEventListener('focus', handleFocus);
+useremail.addEventListener('blur', handleBlur);
+message.addEventListener('input', handleInput);
```

Comment:
![Screen Shot 2021-02-03 at 11 21 06 AM](https://user-images.githubusercontent.com/32880187/106810882-65eec480-6622-11eb-9795-f5e5568dec39.png)
You are missing some event listeners here. Every form control element should have an event listener for `blur`, `focus`, and `input`.

Reply:
![Screen Shot 2021-02-03 at 6 47 07 PM](https://user-images.githubusercontent.com/69738204/106824279-85044b00-6650-11eb-82ce-9e3f016cc789.png)